service/lxd.go

```diff
@@ -73,14 +75,32 @@ func (s LXDService) remoteClient(secret string, address string, port int64) (lxd
 		return nil, err
 	}
 
+	// Use the cluster certificate if none is provided.
+	if cert == nil {
+		clusterCert, err := s.m.FileSystem.ClusterCert()
+		if err != nil {
+			return nil, err
+		}
+
+		cert, err = clusterCert.PublicKeyX509()
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	publicKeyBlock := pem.Block{
+		Type:  "CERTIFICATE",
+		Bytes: cert.Raw,
+	}
```

Comment:
you can use the microcluster `types.X509Certificate{Certificate: cert}.String()` so that you don't have to hard-code this.

Reply:
Thanks, wasn't aware of this utility in microcluster :)

Reply:
Fixed.